modules/nw-metallb-configure-bgp-advertisement.adoc

```diff
@@ -0,0 +1,164 @@
+:_content-type: PROCEDURE
+[id="nw-metallb-configure-BGP-advertisement_{context}"]
+= Configuring MetalLB with a BGP advertisement
+
+Configure MetalLB as follows so that the `IPAddressPool` is advertised with the BGP protocol.
+
+.Prerequisites
+
+* Install the OpenShift CLI (`oc`).
+
+* Log in as a user with `cluster-admin` privileges.
+
+== Example: Advertise a basic address pool configuration with BGP
```

Comment:
Missing anchor ID in heading per https://github.com/openshift/openshift-docs/blob/main/contributing_to_docs/doc_guidelines.adoc#anchoring-titles-and-section-headings